app/repositories/section_repository.rb

```diff
@@ -1,19 +1,10 @@
 class SectionRepository
   NotFoundError = Module.new
 
-  NO_DEFAULT_GIVEN = Object.new
-
-  def fetchable_fetch(key, not_found_value = NO_DEFAULT_GIVEN)
-    if not_found_value != NO_DEFAULT_GIVEN && block_given?
-      raise ArgumentError.new("Cannot provide both a default arg and block to #fetch")
-    end
-
+  def fetchable_fetch(key)
     result = public_send(:[], key)
 
     if result.nil?
-      return yield(key) if block_given?
```

Comment:
This is removed in the commit with subject "Simplify SectionRepository#fetchable_fetch".